cmake/Modules/FindComputeCpp.cmake

```diff
@@ -232,13 +238,18 @@ endfunction()
 #  target and sets a dependancy on that new command.
 #
 #  targetName : Name of the target to add a SYCL to.
-#  sourceFile : Source file to be compiled for SYCL.
 #  binaryDir : Intermediate directory to output the integration header.
+#  sourceFiles : Source files to be compiled for SYCL.
 #
-function(add_sycl_to_target targetName sourceFile binaryDir)
+function(add_sycl_to_target targetName binaryDir sourceFiles)
```

Comment:
I don't have any other ideas, I suspected that it was the varargs thing causing the change. If you're happy to alter the other CMake files to fit the change I'd be very grateful, though I'd be happy to fix it up as part of the merge - whatever works for you! :) (My CMake-fu is not very strong)